### PR TITLE
fix: [Spore] swap polish

### PR DESCRIPTION
**Pull request by callil**
## Description
Update visual nits on the swap widget based on design feedback.
- Token selector button sizing and border radius
- Slippage warning UI (adds simple logic to show/hide an icon)
- Swap review modal font weight
- Settings input focused state
- Warning label sizing and icon

## Screen capture

|    Before    |   After    |
| ------------ | ------------ |
| Screenshot 2023-09-11 at 12 08 39 PM | Screenshot 2023-09-11 at 12 09 03 PM |
| Screenshot 2023-09-11 at 12 09 52 PM | Screenshot 2023-09-11 at 12 10 08 PM |
| Screenshot 2023-09-11 at 12 11 40 PM | Screenshot 2023-09-11 at 12 12 30 PM |
| Screenshot 2023-09-11 at 12 14 02 PM | Screenshot 2023-09-11 at 12 13 59 PM |

### CODE REVIEW

**Comment by natew** [comment]
@callil this has some overlap with my other pr:

https://github.com/Uniswap/interface/pull/7240

i will check to see but maybe just that token selector. if you want to sign off on the visual changes there we can merge it and then i can take over this pr and make sure it works together well.

File: src/components/Settings/MenuButton/index.tsx [hunk 1]
```diff
@@ -55,10 +58,15 @@ const ButtonContent = () => {
     )
   }
 
+  // const tooLow = userSlippageTolerance !== userSlippageTolerance.lessThan(MINIMUM_RECOMMENDED_SLIPPAGE)
```
Comment (natew) [inline src/components/Settings/MenuButton/index.tsx:61]:
```suggestion
```

File: src/components/swap/SwapModalHeaderAmount.tsx [hunk 2]
```diff
@@ -1,61 +1,73 @@
...
-import { MouseoverTooltip } from 'components/Tooltip'
-import { useWindowSize } from 'hooks/useWindowSize'
-import { PropsWithChildren, ReactNode } from 'react'
-import { TextProps } from 'rebass'
-import { Field } from 'state/swap/actions'
-import styled from 'styled-components'
-import { BREAKPOINTS, ThemedText } from 'theme'
-import { NumberType, useFormatter } from 'utils/formatNumbers'
-import { formatReviewSwapCurrencyAmount } from 'utils/formatNumbers'
+import { Currency, CurrencyAmount } from "@uniswap/sdk-core";
```
Comment (natew) [inline src/components/swap/SwapModalHeaderAmount.tsx:1]: this just needs formatter to run

**Review by natew — APPROVED** [review]
just needs formatter to run and remove that commented out line and it lgtm

File: src/components/swap/SwapModalHeaderAmount.tsx (hunk 2 above)
Comment (callil) [reply src/components/swap/SwapModalHeaderAmount.tsx]: Done

File: src/components/Settings/MenuButton/index.tsx [hunk 3]
```diff
@@ -31,19 +33,20 @@ const Button = styled.button<{ isActive: boolean }>`
...
 
 const IconContainerWithSlippage = styled(IconContainer)<{ displayWarning?: boolean }>`
   div {
-    color: ${({ theme, displayWarning }) => (displayWarning ? theme.deprecated_accentWarning : theme.neutral2)};
+    color: ${({ theme, displayWarning }) => (displayWarning ? theme.neutral2 : theme.neutral2)};
   }
 
-  background-color: ${({ theme, displayWarning }) =>
-    displayWarning ? theme.deprecated_accentWarningSoft : theme.surface2};
+  background-color: ${({ theme, displayWarning }) => (displayWarning ? theme.surface2 : theme.surface2)};
```
Comment (natew) [inline src/components/Settings/MenuButton/index.tsx:45]: This conditional has the same value on each side can  be removed altogether

**Review by natew — CHANGES_REQUESTED** [review]
one more small fix

File: src/components/Settings/MenuButton/index.tsx (hunk 3 above)
Comment (callil) [reply src/components/Settings/MenuButton/index.tsx]: fixd

**Review by just-toby — APPROVED** [review]

File: src/components/Settings/MenuButton/index.tsx [hunk 4]
```diff
@@ -55,10 +57,12 @@ const ButtonContent = () => {
     )
   }
 
-  const isInvalidSlippage = validateUserSlippageTolerance(userSlippageTolerance) !== SlippageValidationResult.Valid
+  const tooHigh = userSlippageTolerance.greaterThan(MAXIMUM_RECOMMENDED_SLIPPAGE)
```
Comment (JFrankfurt) [inline src/components/Settings/MenuButton/index.tsx:60]: If I'm reading this correctly, it looks like you're removing the only current use of `validateUserSlippageTolerance`. Are you sure we want to remove the slippage too low case here?

```
  if (userSlippageTolerance.lessThan(MINIMUM_RECOMMENDED_SLIPPAGE)) {
    return SlippageValidationResult.TooLow
  }
```

Comment (JFrankfurt) [reply src/components/Settings/MenuButton/index.tsx:60]: If this is intended and it is the last usage of the function, then we should remove the file `src\utils\validateUserSlippageTolerance.ts`

**Pull request closed without merging**